runtime/src/test/java/com/dylibso/chicory/runtime/ModuleTest.java

```diff
@@ -321,6 +321,17 @@ public void issue294_BRTABLE() {
         assertEquals(4, main.apply()[0].asInt());
     }
 
+    @Test
+    public void fuzzCrashReproY4ye0lx() {
```

Comment:
I'm confused. What is this supposed to test? Since all the tests pass, it's not crashing.

Reply:
This test demonstrates that the interpreter is emitting the correct result, and the problem is in "mimicking" the `wasmtime` output.
I will follow up on this eventually, I'm considering moving to `wasm-interp` as an "Oracle" as it has a better handling of Wasm types.